joplin/base/wagtail_hooks.py

```diff
@@ -10,46 +10,6 @@
 from base.models import HomePage, Topic, Location, Contact
 
 
-@hooks.register('insert_global_admin_css')
```

Comment:
So we're removing the asset inserting here in the py files and the new code is what happens with `{% render_bundle 'bundle_name' %}` in the html files?

It seems like those bundles right now get collected at the entrypoints. Do those have to be js files? What if all the customization we are doing to a template is in the css? I think I'm missing a step here in my understanding

Reply:
For now I think the best path would be to create a .js file that just imports the .scss files.